esp/templates/contact_success.html

```diff
@@ -1,9 +1,9 @@
 {% extends "main.html" %}
 
-{% block title %}Contact ESP{% endblock %}
+{% block title %}Contact {{ settings.INSTITUTION_NAME }} {{ settings.ORGANIZATION_SHORT_NAME }}{% endblock %}
```

Comment:
Here it might read better to just use the short name, not the institution name.  "Contact MIT ESP" reads a little weirdly to me if that's the only program you ever interact with.

Reply:
Okay, I don't get the same weirdness, but I trust your judgment